source/_docs/energy/electricity-grid.markdown

```diff
@@ -45,6 +45,11 @@ We have developed [Home Assistant Glow](https://github.com/klaasnicolaas/home-as
 
 ![Photo of Home Assistant Glow attached to an electricity meter](/images/docs/energy/home-assistant-glow.jpg)
 
+#### Read the meter using an AI-on-the-edge-device
+[AI-on-the-edge-device](https://github.com/jomjol/AI-on-the-edge-device) is a project running on an ESP32-CAM and can be fully integrated into Home Assistant using the Home Assistant Discovery Functionality resp. MQTT. It digitalizes your gas/water/electricity meter display and provides its data through various ways. 
+
+![image](https://user-images.githubusercontent.com/1783586/201498709-fafd7f64-d131-42c9-bf89-3d990cbadf33.png)
```

Comment:
- Images need to be included in this repository, we don't allow external images.
- Please provide a meaningful alt description.

Reply:
Thanks for looking into it.

I adjusted it according your review.

I could not figure out how to test if the image URL is correct, so I just made it like the other images on that page.